index.js

```diff
@@ -103,16 +120,19 @@ exports.single_recipient = function (next, connection) {
 
   transaction.results.add(this, { fail: 'single_recipient', emit: true })
 
-  if (!this.cfg.reject.single_recipient) return next()
+  if (this.cfg.reject.single_recipient) {
+    next(DENY, 'this bounce message has too many recipients')
```

Comment:
```suggestion
    return next(DENY, 'this bounce message has too many recipients')
```

You do need the return here, for flow control. Otherwise you'll be calling `next()` twice.